src/main/resources/synthetic.xml

```diff
@@ -27,8 +27,9 @@
 
     <type type="jira.CheckIssue" extends="jira.JiraScript">
         <property name="issueId" category="input"/>
-        <property name="expectedStatus" category="input" default=""/>
+        <property name="expectedStatusList" category="input" kind="list_of_string"  description="List one or more expected statuses to check for" default=""/>
```

Comment:
The empty string default strikes me as odd here, but I see how that's carried over from how it was.

Reply:
Yeah, I agree. I wasn't sure if it would cause a problem by removing it.